test/test_foreach.py

```diff
@@ -3,6 +3,8 @@
 from torch.testing._internal.common_utils import TestCase, run_tests
 from torch.testing._internal.common_device_type import instantiate_device_type_tests, dtypes, skipCUDAIfRocm
 
+N_values = [30]
```

Comment:
@izdeby do you know how `[30, 300]` values have been picked originally?
```suggestion
N_values = [20, 50]
```

Reply:
yes. these values represent the sizes which fit into 1 batch for multi tensor apply and dont. we definitely want both values.